OxfordBodleian/Juel-Jensen/BDLaethg28.xml

```diff
@@ -74,45 +81,48 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </support>
                                 <extent>
                                     <measure unit="leaf">16</measure>
-                                    <measure unit="leaf" type="blank"></measure>
-                                    <!--<locus target=""/>-->
                                     <dimensions type="outer" unit="mm">
                                         <height>105</height>
+                                        <!--On the metadata is written: 104?-->
                                         <width>75</width>
                                         <depth>23</depth>
+                                        <!--On the metadata is written: 18?-->
                                     </dimensions>
                                     <dimensions type="folio" unit="mm">
                                         <height>103</height>
                                         <width>72</width>
                                     </dimensions>
-                                    <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#9r"/>.</note>
+                                    <note corresp="#leafdim">Data on leaves dimensions taken from
+                                            <locus target="#9r"/>.</note>
                                     <measure type="weight" unit="g">84</measure>
                                 </extent>
-                                <foliation>Pencil foliation in the upper right corner carried out by the Bodleian Library during our stay 
-                                in January 2020.</foliation>
+                                <foliation>The manuscript is not foliated.</foliation>
```

Comment:
It was foliated after we took the the photos

Reply:
Aha! I'll change it to the previous statement